Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM registry.fedoraproject.org/fedora:32
+RUN dnf --assumeyes --refresh group install "Development Tools" "Development Libraries"
+RUN dnf clean all
+ADD . /cinder
+RUN /cinder/oss-build-and-test.sh
```

Comment:
Can you remove this line? Assuming the intent of the Dockerfile is to give users a development container that has all the dependencies necessary to build Cinder, that's all it should do.